### PR TITLE
Add jq command line utility for parsing JSON

### PR DESCRIPTION
**Pull request by rahmanqureshi**
jq is a command line tool for parsing JSON files.

### CODE REVIEW

**Review by tfoote — CHANGES_REQUESTED** [review]
Could you add debian and fedora rules too. They're all the same key.

https://packages.ubuntu.com/xenial/jq
https://packages.debian.org/buster/jq
https://admin.fedoraproject.org/pkgdb/package/rpms/jq/

**Comment by Alessandro-Barbieri** [comment]
`app-misc/jq` on gentoo

File: rosdep/base.yaml [hunk 1]
```diff
@@ -1142,6 +1142,11 @@ joystick:
   opensuse: [input-utils]
   rhel: [joystick]
   ubuntu: [joystick]
+jq:
+  debian: [jq]
+  fedpra: [jq]
```
Comment (Alessandro-Barbieri) [inline rosdep/base.yaml]: fedora

**Review by Alessandro-Barbieri — CHANGES_REQUESTED** [review]

**Comment by RahmanQureshi** [comment]
Done!

**Comment by RahmanQureshi** [comment]
Oops, done!

**Review by mikaelarguedas — APPROVED** [review]
lgtm

**Pull request merged**